data/translations/kdiskmark_cs_CZ.ts

```diff
@@ -0,0 +1,441 @@
+    <message>
+        <location filename="../../src/benchmark.cpp" line="201"/>
+        <location filename="../../src/benchmark.cpp" line="217"/>
+        <source>Random Read</source>
+        <translation>Náhodné čtení</translation>
+    </message>
+    <message>
+        <location filename="../../src/benchmark.cpp" line="209"/>
+        <location filename="../../src/benchmark.cpp" line="225"/>
+        <source>Random Write</source>
+        <translation>Náhodný zápis</translation>
+    </message>
+    <message>
+        <location filename="../../src/benchmark.cpp" line="235"/>
+        <source>Interval Time %1/%2 sec</source>
+        <translation>Prodleva mezi testy %1/%2 sek</translation>
+    </message>
+</context>
+<context>
+    <name>MainWindow</name>
+    <message>
+        <location filename="../../src/mainwindow.ui" line="14"/>
+        <source>KDiskMark</source>
+        <translation>KDiskMark</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.ui" line="30"/>
+        <location filename="../../src/mainwindow.cpp" line="417"/>
+        <source>All</source>
+        <translation>Vše</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.ui" line="108"/>
+        <location filename="../../src/mainwindow.ui" line="158"/>
+        <location filename="../../src/mainwindow.ui" line="228"/>
+        <location filename="../../src/mainwindow.ui" line="252"/>
+        <location filename="../../src/mainwindow.ui" line="289"/>
+        <location filename="../../src/mainwindow.ui" line="313"/>
+        <location filename="../../src/mainwindow.ui" line="350"/>
+        <location filename="../../src/mainwindow.ui" line="387"/>
+        <source>%v</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.ui" line="188"/>
+        <source>MB/s</source>
+        <translation>MB/s</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.ui" line="193"/>
+        <source>GB/s</source>
+        <translation>GB/s</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.ui" line="198"/>
+        <source>IOPS</source>
+        <translation>IOPS</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.ui" line="203"/>
+        <source>μs</source>
+        <translation>μs</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.ui" line="134"/>
+        <source>SEQ 1</source>
+        <translation>SEQ 1</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.ui" line="265"/>
+        <source>SEQ 2</source>
+        <translation>SEQ 2</translation>
+    </message>
+    <message>
+        <location filename="../../src/mainwindow.ui" line="326"/>
+        <source>RND 1</source>
+        <translation>NHD 1</translation>
```

Comment:
That's not good idea to translate RND

Reply:
Maybe you are right - tried to be comprehensible there, but must admin, that this abbreviation is not used somewhere else. Thus reverting to English one.